benchmarks/callbacks/benchmark_RegionTimer.cpp

```diff
@@ -0,0 +1,246 @@
+#include <iostream>
+
+#include "benchmark/benchmark.h"
+
+#include "Kokkos_Core.hpp"
+#include "Kokkos_Graph.hpp"
+
+#include "kokkos-utils/callbacks/EventNameMatcher.hpp"
+#include "kokkos-utils/callbacks/Manager.hpp"
+#include "kokkos-utils/callbacks/SequenceOfRegionTimer.hpp"
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+#define REGION_LABEL "This is a quite long region name, hopefully it's not using SBO at all."
+
+namespace Kokkos::utils::benchmarks::callbacks
+{
```

Comment:
```suggestion
namespace Kokkos::utils::benchmarks::callbacks
{

using namespace Kokkos::utils::callbacks;
```
(to avoid `Kokkos::utils::callbacks::` everywhere) (we do it already in the tests)

Reply:
I prefer keep it here.